src/lib/style/abstracts/_mixins.scss

```diff
@@ -77,10 +77,15 @@
     transition: border-bottom 300ms ease-in;
   }
   > *[aria-selected='true'] {
-    border-bottom: 3px solid SRC.$primary-action-color-500;
+    border-bottom: 3px solid $active-color;
+    color: #22252A;
+    svg {
+      color: SRC.$primary-action-color-500;
+    }
   }
   > *[aria-selected='false'] {
-    border-bottom: 3px solid lightgray;
+    color: #878E95;
```

Comment:
Are these hard-coded colors (#22252A and #878E95) specified by design, and if so, can we assign them to named variables?

Reply:
Taken from the design, will pull out into a named variable.